### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by ghanatava** [coredns/coredns#6359] [Q&A] [answered]
Hey! I am learning Kubernetes for a while  and I am really interested in coredns project. How can I start out with the project , can anyone suggest me some learning resources which can help me getting started

**Comment by chrisohaver** [comment, marked as answer]
There is the Learning CoreDNS book written by one of our maintainers, @johnbelamaric ... https://www.oreilly.com/library/view/learning-coredns/9781492047957/

**Comment by ghanatava** [comment]
Thank you, I will look into it.
